Dockerfile

```diff
@@ -2,6 +2,13 @@
 ### The image itself is mainly used as a container for the binaries and for starting e2e tests with custom parameters.
 ### By default, the binaries inside the image have some mock parameters and can start, but are not intended to be used
 ### inside this image in the real deployments.
+###
+### Local build instructions:
+### 1. Clone the `neondatabase/build` repo somewhere. Recommended location is `../build`.
```

Comment:
<img width="346" alt="image" src="https://user-images.githubusercontent.com/2690773/216603619-2953d640-6e01-4506-8b60-31e1deb031bb.png">

That would be amazing to do indeed, but alas.